src/hooker/setupglobals_zh.cpp

```diff
@@ -55,7 +55,7 @@ uint32_t &View::s_idNext = Make_Global<uint32_t>(PICK_ADDRESS(0x009D4178, 0));
 
 // water.cpp
 #include "water.h"
-WaterSetting *g_waterSettings = Make_Pointer<WaterSetting>(PICK_ADDRESS(0x00A2F0B8, 0x00A2F0B8));
+ARRAY_DEF(PICK_ADDRESS(0x00A2F0B8, 0x00A2F0B8), WaterSetting, g_waterSettings, 5);
```

Comment:
Can use `TIME_OF_DAY_COUNT` here too?

Btw, why are these Macros necessary? Can this not be typed out simply without the macros or would that look too complicated?

Reply:
The macros are not necessary and i can remove them I initially didn't do that as I wanted to ensure it would work with minimal changes.